### ISSUE TITLE
Inactive tab text color is not accessible

### ISSUE THREAD

**Issue by dgtlntv** [canonical/maas-discourse-theme#11]
The very light gray of non-active tabs does not meet accessibility requirements. We should change the color of the currently inactive tab text to something with more contrast.
![Screenshot 2024-12-13 at 09 07 40](https://github.com/user-attachments/assets/4319faa1-0d65-4987-87c8-e2b3726c7a32)
![Screenshot 2024-12-13 at 09 09 38](https://github.com/user-attachments/assets/6946c11e-9b1f-4326-a5e4-395a92a40bc7)

**Comment by dgtlntv** [comment]
On closer inspection, the dark area should probably extend further so that it also covers the discourse menu. This would provide an accessible contrast for these links.
